qa/common/infer_util.py

```diff
@@ -52,30 +61,27 @@ def _range_repr_dtype(dtype):
         return np.int32
     return dtype
 
-def _prepend_string_size(input_values):
-    input_list = []
-    for input_value in input_values:
-        input_list.append(serialize_string_tensor(input_value))
-    return input_list
 
 # Perform inference using an "addsum" type verification backend.
 def infer_exact(tester, pf, tensor_shape, batch_size,
                 input_dtype, output0_dtype, output1_dtype,
                 output0_raw=True, output1_raw=True,
-                model_version=None, swap=False,
+                model_version="", swap=False,
                 outputs=("OUTPUT0", "OUTPUT1"), use_http=True, use_grpc=True,
-                skip_request_id_check=False, use_streaming=True,
+                use_json=True, skip_request_id_check=False, use_streaming=True,
```

Comment:
use_http_json_tensors would be more descriptive

Reply:
Done